addons/sale/tests/test_credit_limit.py

```diff
@@ -0,0 +1,50 @@
+from odoo.addons.sale.tests.common import SaleCommon
+from odoo.fields import Command
+from odoo.tests import tagged
+
+
+@tagged('post_install', '-at_install')
+class TestSaleOrder(SaleCommon):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+
+        cls.env.company.account_use_credit_limit = True
+
+        buck_currency = cls.env['res.currency'].create({
+            'name': 'Test Buck ',
+            'symbol': 'TB',
+            'rounding': 0.01,
+            'position': 'after',
+            'currency_unit_label': 'TestBuck',
+            'currency_subunit_label': 'MockCent',
+        })
```

Comment:
```suggestion
        buck_currency = cls.env['res.currency'].create({
            'name': 'Test Buck ',
            'symbol': 'TB',
        })
```
We don't really need the other fields for now (we might add them later if needed) :wink: